exquery-restxq-specification/restxq-1.0-specification.html

```diff
@@ -886,6 +886,8 @@ <h4>rest:uri()</h4>
         </section>
       </section>
     </section>
+    
+    <section id="conformance"/>
```

Comment:
I think this is missing some additional conformance information... How about something like:

```html
    <section id="conformance">
      <p>This section defines the conformance criteria for a RESTXQ 1.0 implementation. An implementation that claims to conform to this specification MUST include a claim of Minimal Conformance as defined in <a href="#minimal-conformance" class="sectionRef"></a>.</p>


      <section id="minimal-conformance">
        <h3>Minimal Conformance</h3>
        <p>An implementation that claims <strong>Minimal Conformance</strong> to this specification MUST provide all of the following items:</p>
        <ol>
          <li><p>An implementation of everything specified in this document.</p></li>
          <li><p>A definition of every item specified to be implementation-defined.</p></li>
        </ol>
      </section>
    </section>
```

Reply:
Happy to add this; just wanted to note that the empty section is automatically expanded by respec with the standard text:

> As well as sections marked as non-normative, all authoring guidelines, diagrams, examples, and notes in this specification are non-normative. Everything else in this specification is normative.
>
> The key words MAY, MUST, MUST NOT, and SHOULD in this document are to be interpreted as described in [BCP 14](https://datatracker.ietf.org/doc/html/bcp14) [[RFC2119](file:///Users/yamahito/Projects/exquery/exquery-restxq-specification/restxq-1.0-specification.html#bib-rfc2119)] [[RFC8174](file:///Users/yamahito/Projects/exquery/exquery-restxq-specification/restxq-1.0-specification.html#bib-rfc8174)] when, and only when, they appear in all capitals, as shown here.

(which will be pre-pended to the result)

Reply:
Yes, I did test it, please add it.

Reply:
Already added below: https://github.com/exquery/exquery/pull/151/commits/1ebc73cb5c322cd715887d924f08d4b066597857